csharp/src/Ice/Internal/ThreadPool.cs

```diff
@@ -333,9 +333,9 @@ public void executeFromThisThread(System.Action call, Ice.Connection connection)
             }
             catch (System.Exception ex)
             {
-                if (_instance.initializationData().properties.getIcePropertyAsInt("Ice.Warn.Dispatch") > 0)
+                if (_instance.initializationData().properties.getIcePropertyAsInt("Ice.Warn.Executor") > 0)
                 {
-                    _instance.initializationData().logger.warning("dispatch exception:\n" + ex);
+                    _instance.initializationData().logger.warning($"executor exception:\n{ex}");
```

Comment:
As of Ice 3.8, dispatch has the same meaning in Ice and IceRPC: the server-side processing of a request.